docs/src/main/sphinx/release/release-356.md

```diff
@@ -0,0 +1,80 @@
+# Release 356 (XX Apr 2021)
+
+## General
+
+* Add support for {doc}`/sql/match-recognize`. ({issue}`6111`)
+* Add {func}`soundex` function. ({issue}`4022`)  
+* Introduce `system.metadata.materialized_view_properties` table for listing available materialized view properties. ({issue}`7615`)
+* Introduce planning time limit which can be specified using `query.max-planning-time` config property. ({issue}`7213`)
+* Add configuration for the query info sent to clients.  This allows clients to be directed to a different location for 
+  information about their query.  This is set using the `query.info-url-template` property.  ({issue}`7678`)
+* Improve cancellation of queries during planning phase. ({issue}`7213`)
+* Improve performance of `ORDER BY ... LIMIT` queries over a `LEFT JOIN`. ({issue}`7028`)
```

Comment:
> but we chose Top N pushdown as name for such clauses

No, use use `ORDER BY ... LIMIT` for that. Top N doesn't mean anything in SQL -- it's just an implementation detail of the engine and internal APIs. Referring to it as `ORDER BY ... LIMIT` is the right thing to put it in context of the SQL someone might write.

Reply:
We call it Top-N pushdown in https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/optimizer/pushdown.rst#top-n-pushdown to have a convenient way to refer to it, since trying to use SQL syntax in all the links doesn't really work. That documentation explains why it is called that and gives both syntax forms.

However, for this release note, it's actually not about pushdown, so using the syntax makes sense.